travis-ci-build.sh

```diff
@@ -3,4 +3,12 @@ set -ex
 
 npm run lint
 npm test
+
+./generate.sh
+MODIFIED_FILES=$(git diff --name-only)
+if [[ -n $MODIFIED_FILES ]]; then
+  echo "ERROR: Changes detected in generated code, please run './generate.sh' and check-in the changes."
```

Comment:
This will unfortunately be invisible to our users and hence they won't be able to self-fix this. Not sure if we have a fix for this. We could make the project visible but not modifiable to the public?